CMakeLists.txt

```diff
@@ -85,6 +85,26 @@ endif(WITH_CCACHE)
 
 option(WITH_SCCACHE "Build with sccache.")
 if(WITH_SCCACHE)
+  if(NOT FORCE_MAX_JOBS)
+    execute_process(
+      COMMAND "sccache" "--dist-status"
+      OUTPUT_VARIABLE sccache_dist_status
+      RESULT_VARIABLE sccache_dist_status_result
+      COMMAND_ERROR_IS_FATAL ANY
+    )
+    message(STATUS "sccache --dist-status: ${sccache_dist_status}")
```

Comment:
i'd suggest drop this line. it's useful for debugging. but IMHO, we should not have it when we are not debugging this feature.